src/main/java/me/totalfreedom/totalfreedommod/command/Command_vanish.java

```diff
@@ -0,0 +1,70 @@
+package me.totalfreedom.totalfreedommod.command;
```

Comment:
You should probably also update the /list command, as invis players should still show in that because TFM will not check if they are invis currently.